packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -317,6 +319,22 @@ export class ResourceService {
     });
   }
 
+  async messageBrokerConnectedServices(args: FindOneArgs): Promise<Resource[]> {
+    const resource = await this.resource(args);
+    const serviceTopicsCollection = await this.serviceTopicsService.findMany(
+      {}
```

Comment:
filter serviceTopics by relation to the current project only - you are now using all serviceTopic in the system from any project and any workspace

Reply:
Done